src/MOI/MOI_wrapper.jl

```diff
@@ -2907,13 +2907,33 @@ function MOI.optimize!(model::Optimizer)
         _set_MIP_start(model)
     end
     start_time = time()
-    if model.has_nlp_constraints
-        @checked Lib.XPRSnlpoptimize(model.inner, model.solve_method)
-    elseif is_mip(model)
-        @checked Lib.XPRSmipoptimize(model.inner, model.solve_method)
+
+    version = getversion()
```

Comment:
This is absolutely zero overhead?
Xpress sometimes is impressively slow for very simple things.

If no we could run this on initialization and save in a const.

Reply:
```
@btime Xpress.getversion()
  715.517 ns (11 allocations: 1.64 KiB)
v"44.1.1"
```
Does not seem to be a significant overhead.